src/build.rs

```diff
@@ -96,29 +97,20 @@ impl Build {
     }
 
     /// Use `llvm-config --version` command
-    pub fn version(&self) -> Result<(u32, u32, u32)> {
+    pub fn version(&self) -> Result<Version> {
         let (stdout, _) = Command::new(self.prefix().join("bin/llvm-config"))
             .arg("--version")
             .check_output()?;
         parse_version(&stdout)
     }
 }
 
-fn parse_version(version: &str) -> Result<(u32, u32, u32)> {
+fn parse_version(version: &str) -> Result<Version> {
     let cap = Regex::new(r"(\d+).(\d).(\d)")
```

Comment:
I realize this was here before, but since I'm freshly looking at this code:  is this regex correct?  It seems like it would capture, *e.g.*, `123+456y0` because the `.`’s are unescaped.  Also, since the altered logic doesn't require the capturing groups, can you just get rid of them?